docs/concepts/sensor-models/sensor-models.md

```diff
@@ -0,0 +1,90 @@
+# Sensor Models
+
+This document is intended to provide information related to sensor models by presenting an introduction to sensor models, a brief explanation of the Community Sensor Model (CSM), and an overview of several ASC software packages that directly relate to the instantiation and exploitation of sensor models.
+
+## Introduction to Sensor Models
```

Comment:
might be worth mentioning what DNs are and how they relate to sensor models. Knowing that the number of lines and samples are important but not the DNs themselves I think is important context.

Reply:
A DN isn't a sensor model though. As a whole, the observation isn't a sensor model. They are conceptually two different things. I would suggest some ordering of docs like:

1. Reference Frames
2. Sensor Models
3. Observations (talk about DNs in here)
4. Bringing it all together through an example: surface images, mathematical sensor model, data in the correct place, etc.

Reply:
>  Knowing that the number of lines and samples are important but not the DNs themselves I think is important context.

> A DN isn't a sensor model though. As a whole, the observation isn't a sensor model. They are conceptually two different things.

yes, that do be what I am saying and putting that distinction here is important.